### PR TITLE
Remove border radius from chat

### PR DESCRIPTION
**Pull request by chapoi**
With not-so-mild heartache stripping the default border-radius from chat to be more conform standard Discourse.

Introduced another variable to this end too so we have:
```
--d-border-radius
--d-border-radius-large
```

Whenever I work with radius, I usually have a 4px from small elements like buttons, and a 8px for large elements like cards, so that they visually "feel" the same.

Having these 2 vars makes it easy to at style chat again with rounded borders.

### CODE REVIEW

**Comment by jjaffeux** [comment]
I can approve it but probably would better for designer @jordanvidrine

**Comment by chapoi** [comment]
> I can approve it but probably would better for designer @jordanvidrine

Since Jordan is out, maybe @awesomerobot can help

File: app/assets/stylesheets/common/foundation/base.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
   --topic-body-width-padding: #{$topic-body-width-padding};
   --topic-avatar-width: #{$topic-avatar-width};
   --d-border-radius: initial;
+  --d-border-radius-large: initial;
```
Comment (jordanvidrine) [inline app/assets/stylesheets/common/foundation/base.scss:10]: So this is essentially saying:

"No border radius, unless specified in a theme/theme-component?

**Review by jordanvidrine — APPROVED** [review]

File: app/assets/stylesheets/common/foundation/base.scss (hunk 1 above)
Comment (chapoi) [reply app/assets/stylesheets/common/foundation/base.scss:10]: Yup, that's the idea. Good?

**Pull request merged**